docs/modules/ROOT/pages/explanation/decisions/zone-naming.adoc

```diff
@@ -0,0 +1,34 @@
+= Zone naming scheme
+
+Problem::
+{product} will consist of several clusters we refer to as zones.
+We need a common understanding how we name them.
+
+Alternatives::
+* Using a randomly generated identifier like `rapid-sound-0933` or `rapid-sound` (`rapid-sound.appuio.cloud`).
+  We have tried that already with https://syn.tools[Project Syn].
+  This is way better than for example a random string of characters or a UUID.
+  From the perspective of daily operation, we have learned that this was still too much mental load even for a small number of items.
+* Use top level domain to encode country locality (`<zone>.appuio.ch`, `<zone>.appuio.de`).
+  This does not work as not every country has a country specific top level domain (for example the United States of America).
+* Use the cloud provider followed by a numerical or alphabetical index (`cloudscale-a.appuio.cloud` or `exoscale-1.appuio.cloud`).
+* Use the country code followed by a numerical or alphabetical index (`ch-1.appuio.cloud` or `de-a.appuio.cloud`).
+
+Decision::
+The zone identifier will be built following the pattern:
++
+****
+`<provider>-<region>-<index>`
+****
++
+`<provider>`::: Name of the cloud provider the cluster is hosted on (`cloudscale`, `exoscale`, `aws`, `azure`, `gcp`, etc.).
+`<region>`::: Region name, taken verbatim from the cloud providers own naming scheme (`rma`, `ch-dk-2`, `eu-central-1`, `switzerlandnorth`, `europe-west6`).
+`<index>`::: Zero indexed numerical identifier.
```

Comment:
That is way I brought up in the examples. Reading it from you, made me tend to the alphabetical index.

Reply:
What speaks pro alphabetical is that there are 26 characters instead of 10 for one single char. Also the sort algorithm is less issues with. But since we have this random in the cloud provider naming as well, I think at the end it doesn't matter so much. Open to both.

Reply:
The sorting really depends on the algorithm. Even with alphabetical characters, system sort "unexpectedly":
```
a
aa
aab
b
bb
```
So, in the unlikely event we have more than 26 clusters in a region, "aa" would still be listed second, even if it's actually the 27th cluster. Like sorting of numbers:
```
1
11
2
```
So, without pre-zeroes like `01, 02`, I would just ignore the sorting of the indexes...

Reply:
As I said. I do not have a strong opinion here. Fair enough just to keep it as is.